Paws4Me/HomeCollectionViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  HomeCollectionViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/04/12.
+//
+
+import Foundation
+import UIKit
+
+class HomeCollectionViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate {
+    let homeImages: [String] = ["dogCV", "catCV", "dog3CV"]
```

Comment:
Maybe you can name this more clearly, like, "All pets", "Favourites", "Adoption Progress".

Reply:
Good suggestion 👍🏻